completions/_nmcli

```diff
@@ -89,23 +82,23 @@ _comp_cmd_nmcli()
             nm)
                 case $command in
                     enable)
-                        _comp_cmd_nmcli__list "true false"
+                        _comp_compgen_split "true false"
```

Comment:
Actually, I think there is no reason. I just replaced the function name because the function `_comp_cmd_nmcli__list` is just the same as `_comp_compgen_split`. I thought that the original author of the `nmcli` completion wanted to be more descriptive by the function name than just calling `compgen -W` directly, but I can remove the use of a helper function here.

Reply:
Applied in [`f9c0804..6df80b0`](https://github.com/scop/bash-completion/compare/f9c0804c12f2125f631dc7ea875aa45cb0eaad2d..6df80b008eb49051cfc4b62c09de4a84051325db) (squashed into 1c48858d).